File: elasticsearch/es-client/src/main/java/me/kzv/elasticsearch/model/Subway.java
package me.kzv.elasticsearch.model;

import com.opencsv.bean.CsvBindByPosition;
import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Setter
@Getter
@ToString
public class Subway {
  @CsvBindByPosition(position = 0)
  private String code;

  @CsvBindByPosition(position = 1)
  private String station;

  @CsvBindByPosition(position = 2)
  private String line;

  @CsvBindByPosition(position = 3)
  private String excode;
}

File: elasticsearch/searching-server/src/main/kotlin/me/kzv/searchingserver/document/Subway.kt
package me.kzv.searchingserver.document

import com.opencsv.bean.CsvBindByPosition

class Subway(
    @field:CsvBindByPosition(position = 0)
    val code: String,

    @field:CsvBindByPosition(position = 1)
    val station: String,

    @field:CsvBindByPosition(position = 2)
    val line: String,

    @field:CsvBindByPosition(position = 3)
    val excode: String
)
